dotcom-rendering/src/web/lib/DecideContainer.tsx

```diff
@@ -90,7 +87,6 @@ export const DecideContainer = ({
 					trails={trails}
 					containerPalette={containerPalette}
 					showAge={showAge}
-					supportingContent={supportingContent}
```

Comment:
This was seemingly used by an older implementation of Sublinks for `fixed/small/slow-IV`, but for some reason it applied the sublink to all of the cards in the container.

This prop was also not used anywhere except the `FixedSmallIVsubLinks.stories.ts` test.